client/src/routes/human_resources/leaves/requests/[id]/edit.svelte

```diff
@@ -0,0 +1,70 @@
+<script context="module">
+  export const preload = ({ params: { id } }) => {
+    return { id };
+  };
+</script>
+
+<script>
+  import { goto, stores } from '@sapper/app';
+  import { _ } from 'svelte-i18n';
+  import { onMount } from 'svelte';
+  import { get, put } from 'utils/axios';
+  import { format } from 'date-fns';
+  import Breadcrumb from 'components/Breadcrumb.svelte';
+  import { errorNormalizer } from 'normalizer/errors';
+  import ServerErrors from 'components/ServerErrors.svelte';
+  import H4Title from 'components/H4Title.svelte';
+  import EditOrUpdateForm from '../_Form.svelte';
```

Comment:
(Nit) Si je compare avec d'autres `edit.svelte` habituellement on appelle ceci `Form`, comme le fichier `_Form.svelte`.

Reply:
j'ai p-e mal compris mais ici ce fichier n'est pas un composant mais une page qui elle utilise le composant _Form.svelte

Reply:
En gros je suggère juste ça

```suggestion
  import Form from '../_Form.svelte';
```